htdocs/js/helpHandler.js

```diff
@@ -57,7 +46,7 @@ $(document).ready(function() {
       div.appendChild(wrap);
       div.appendChild(btn);
 
-      // Markdown format content came from the filesystem and can't
+      // Markdown format content came from the filesystem and can
```

Comment:
I don't think this comment is relevent anymore now that markdown content can come from either the filesystem or database. You can just remove it rather than modify it.